src/main/groovy/com/ullink/Msbuild.groovy

```diff
@@ -240,4 +245,17 @@ class Msbuild extends ConventionTask {
         }
         return cmdParameters
     }
+
+    @OutputDirectories
+    def getOutputDirs() {
+        def paths = []
+        def cmdParameters = getInitProperties()
```

Comment:
AFAIK init properties are optional prop values specified in the gradle build, they may not be provided.
Doubt this works.